src/hooks/UrqlUseSubscriptionWithHandler.re

```diff
@@ -1,38 +1,42 @@
+type handler('a, 'b, 'c) =
```

Comment:
Reading through your comment in #72 (thanks so much for that), could we make these type arguments match the following naming convention?

```suggestion
type handler('acc, 'response, 'ret) =
```

That way it's super clear which type arguments apply to which cases.